nautobot/apps/forms.py

```diff
@@ -53,6 +53,7 @@
 from nautobot.core.forms.widgets import (
     APISelect,
     APISelectMultiple,
+    AutoPopulateWidget,
```

Comment:
Would be good to have an entry in the Added release notes about this, not blocking.